src/c20_server/get_job.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
```

Comment:
This should be the `/get_job` endpoint, not`/`

Reply:
Endpoint changed to /get_job